hardware/TeleinfoBase.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+Domoticz Software : http://domoticz.com/
+File : TeleinfoBase.cpp
+Author : Blaise Thauvin
+Version : 0.1
+Description : This class is used by various Teleinfo hardware decoders to process and display data
+			  It is used by EcoDevices, TeleinfoSerial and more to come
+
+History :
+0.1 2017-03-03 : Creation
+*/
+
+#include "TeleinfoBase.h"
```

Comment:
The first include file in a .cpp file should be
`#include "stdafx.h"`

This will also solve the compile errors